pkgs/tools/networking/vpnc/default.nix

```diff
@@ -1,29 +1,21 @@
-{ lib, stdenv, fetchsvn
-, makeWrapper, pkg-config
-, gawk, gnutls, libgcrypt, nettools, openresolv, perl
+{ lib, stdenv, fetchFromGitHub, fetchpatch
+, makeWrapper, pkg-config, perl
+, gawk, gnutls, libgcrypt, openresolv, vpnc-scripts
 , opensslSupport ? false, openssl # Distributing this is a GPL violation.
 }:
 
 stdenv.mkDerivation {
-  pname = "vpnc";
-  version = "0.5.3-post-r550";
-  src = fetchsvn {
-    url = "https://svn.unix-ag.uni-kl.de/vpnc";
-    rev = "550";
-    sha256 = "0x4ckfv9lpykwmh28v1kyzz91y1j2v48fi8q5nsawrba4q0wlrls";
+  pname = "vpnc-unstable";
+  version = "2021-11-04";
```

Comment:
afaik unstable is added to the version not the pname?

Reply:
Right you are! Fixed. :)

Reply:
I actually realized that the whole version argument is not so clear as I initially thought: https://github.com/NixOS/rfcs/pull/107